library/htmlspecialchars.inc.php

```diff
@@ -40,7 +40,7 @@
  */
 function text($text)
 {
-    return htmlspecialchars($text, ENT_NOQUOTES);
+    return htmlspecialchars($text, ENT_NOQUOTES, 'UTF-8', false);
```

Comment:
We can't hard-code UTF-8. And can I get an example of the double encoding issue?

Reply:
https://community.open-emr.org/t/adding-calendar-categories/9076

you reverted my fix at PR #1091 because double encoding in the add categories screen.  
https://github.com/openemr/openemr/pull/1091/files#diff-20a295b5da1c43c0122a951849451e67R704

Reply:
Regarding double encode;
1. I wouldn't trust that 
2. It will make us lazy (and will end up with some hard to figure out bugs)

Reply:
Do note it's just the odd legacy code that presents these issues, which sometimes require not typical solutions. No reason to pull out this big gun (yet, at least).